src/locales.js

```diff
@@ -26,7 +26,7 @@ export default {
     shortMonthNames: 'Jan_Feb_Mär_Apr_Mai_Jun_Jul_Aug_Sep_Okt_Nov_Dez'.split('_'),
     formatDate: '%b %d, %Y',
     currency: 'EUR',
-    minStay: '%number+ Nächte',
+    minStay: '%number+ nächte',
```

Comment:
I think in german "Nächte" is always written with "N" (capital letter), but my german is as good as my french... so better to ask Yannick 😄

Reply:
nah, `nächte´ is good, you can go on 👍 😄